### DISCUSSION TITLE
Can I change the night or day theme by CLI

### DISCUSSION THREAD

**Discussion by tangjyan** [t1m0thyj/WinDynamicDesktop#542] [Q&A]
![image](https://user-images.githubusercontent.com/21167731/132292616-8f51e71e-bd71-4d86-a9ef-97a3ac9ffaca.png)

I want to change the option, as shown in the picture, by CLI instead of GUI.

**Comment by t1m0thyj** [comment]
As a workaround, you could write a script that toggles the `darkMode` property in the "settings.json" file and restarts WDD.

See [here](https://github.com/t1m0thyj/WinDynamicDesktop/discussions/499) for an example of an AutoHotKey script that does something similar.
